### ISSUE TITLE
pnpm store detected as unlisted binaries

### ISSUE THREAD

**Issue by ChiefORZ** [webpro/knip#208]
Since the upgrade from version 2.19.5 to 2.19.6 knip is detecting pnpm store as a Unlisted binary.

```sh
> knip

Unlisted binaries (1)
store  .github/workflows/test.yml
 ELIFECYCLE  Command failed with exit code 1.
ERROR: "test:knip" exited with 1.
 ELIFECYCLE  Test failed. See above for more details.
```

My github workflow file includes a global available pnpm method, that is used for caching.

```yaml
// .github/workflows/test.yml

- name: Get pnpm store directory
  id: pnpm-cache
  shell: bash
  run: |
    echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT

- uses: actions/cache@v3
  name: Setup pnpm cache
  with:
    path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
    key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
    restore-keys: |
      ${{ runner.os }}-pnpm-store-
```

After i remove these steps, knip will be happy. But this is unexpected.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.19.7. See [Release 2.19.7](https://github.com/webpro/knip/releases/tag/2.19.7) for release notes._

**Comment by webpro** [comment]
Replacing bash-parser with tree-sitter in [v2.19.6](https://github.com/webpro/knip/releases/tag/2.19.6) had the effect of finding more scripts. That revealed the bug in Knip that `store` was still missing as a pnpm sub command.

Thanks for reporting this! Step by step... :)
